x/uibc/errors.go

```diff
@@ -0,0 +1,16 @@
+package uibc
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+var (
+	ErrIBCPauseStatus     = sdkerrors.Register(ModuleName, 1, "invalid ibc pause status")
+	ErrQuotaExceeded      = sdkerrors.Register(ModuleName, 2, "quota transfer exceeded")
+	ErrBadMessage         = sdkerrors.Register(ModuleName, 3, "bad message")
```

Comment:
I think this is not used. Let's delete this.

Reply:
We are using some of the errors from this file

Reply:
yes, but not all